docker-compose.yml

```diff
@@ -67,6 +67,16 @@ services:
     - MYSQL_PASSWORD=mysqldb
     ports:
     - "127.0.0.1:3307:3306"
+    
+  mysql57:
```

Comment:
The two MySql client versions can't access the same server?

Reply:
Nope IICR they changed the login security protocol between versions. But I have to double check that